README.md

```diff
@@ -188,19 +199,19 @@ func tr(key: L10n) -> String {
 Once the code has been generated by the script, you can use it this way in your Swift code:
 
 ```swift
-let title = L10n.AlertTitle.string
+let title = L10n.LocalizableTable.AlertTitle.localized
 // -> "Title of the Alert"
 
 // Alternative syntax, shorter
-let msg = tr(.AlertMessage)
+let msg = tr(L10n.LocalizableTable.AlertMessage)
```

Comment:
If we provide all the overloads necessary for `tr`, is the compiler intelligent enough to find the right enum for `tr(.AlertMessage)` (assuming only one of all the tables have a `case AlertMessage`)? Would be very cool if so, but not sure it's not too much to ask to the compiler…